spring-cloud-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/endpoint/NacosConfigHealthIndicator.java

```diff
@@ -33,15 +31,11 @@
  */
 public class NacosConfigHealthIndicator extends AbstractHealthIndicator {
 
-	private final NacosConfigProperties nacosConfigProperties;
-
 	private final List<String> dataIds;
 
 	private final ConfigService configService;
 
-	public NacosConfigHealthIndicator(NacosConfigProperties nacosConfigProperties,
-			ConfigService configService) {
-		this.nacosConfigProperties = nacosConfigProperties;
+	public NacosConfigHealthIndicator(ConfigService configService) {
 		this.configService = configService;
 
 		this.dataIds = new ArrayList<>();
```

Comment:
dataIds should be remove

Reply:
I think The dataIds we can keep it, Form the dataid list, we can know ,what configuration have we loaded from nacos. private or ext or share.

Reply:
i prefer do it in `NacosConfigEndpoint`, not `NacosConfigHealthIndicator`. how do you think about it?

Reply:
That is a better choice, i will remove it

Reply:
👍